src/oaklib/converters/obo_graph_to_fhir_converter.py

```diff
@@ -97,47 +99,93 @@ def dump(
         self,
         source: GraphDocument,
         target: str = None,
+        code_system_id: str = None,
+        code_system_url: str = None,
         include_all_predicates: bool = True,
+        native_uri_stems: List[str] = None,
+        use_curies_native_concepts: bool = False,
+        use_curies_foreign_concepts: bool = True,
+        predicate_period_replacement: bool = False,
         **kwargs,
     ) -> None:
         """
         Dump an OBO Graph Document to a FHIR CodeSystem.
 
-        By default, only IS_A predicates are converted to ConceptProperties. To override this,
-        specify ``include_all_predicates=True``.
-
-        :param source:
-        :param target:
-        :param include_all_predicates: include the maximal amount of predicates
+        :param source: Source serialization.
+        :param target: Target serialization.
+        :param code_system_id: The code system ID to use for identification on the server uploaded to.
+        See: https://hl7.org/fhir/resource-definitions.html#Resource.id
+        :param code_system_url: Canonical URL for the code system.
```

Comment:
These need to be properly formatted, see https://stackoverflow.com/questions/32566466/multi-line-description-of-a-parameter-description-in-python-docstring

currently results in:

<img width="708" alt="image" src="https://user-images.githubusercontent.com/50745/229572187-047d86b2-ec56-4d2c-878b-9bad988a59d2.png">